### DISCUSSION TITLE
add A5-04-03

### DISCUSSION THREAD

**Discussion by cocoffr** [mak-gitdev/HA_enoceanmqtt#67] [Request] [answered]
hello
can you add device A5-04-03 ?
thx

**Comment by mak-gitdev** [comment, marked as answer]
Hi @cocoffr,

This is already supported. See [Supported Devices](https://github.com/mak-gitdev/HA_enoceanmqtt/wiki/Supported-devices) for more details.

**Comment by mak-gitdev** [comment]
Hi @cocoffr,

Is A5-04-03 working for you ?
If so I can add it as tested and working in the list of supported devices.

**Comment by cocoffr** [comment]
Hi
Yes it's work well!
Thx
